### DISCUSSION TITLE
Add Notifications

### DISCUSSION THREAD

**Discussion by br4yd** [mlemgroup/mormaer-mlem#34] [Ideas]
It would be very useful to receive notifications via the app as you do in the web version. So for e.g. when someone comments your post etc.

**Comment by buresdv** [comment]
This is definitely planned for the (far) future, but it will be very difficult. Notifications are notoriously hard, but I will do my best 🙇 Could you create an [issue](https://github.com/buresdv/Mlem/issues) for this suggestion?

**Reply by br4yd** [reply]
Sure I'll create one
